### PR TITLE
Helm migration 

### PR DESCRIPTION
**Pull request by chandanagcp**

### CODE REVIEW

**Comment by chandanagcp** [comment]
Intentionally failing the `deploy` for now,  to verify the diffs

**Comment by danopia** [comment]
Looking at the diff, we lost the probes. Let's get those back and otherwise LGTM.

Also, the helm CI steps need the `github` context added in addition to the current contexts, for the deploy CI to be green.

**Review by michael-robertson — APPROVED** [review]
Running successfully in sandbox already, cool.
The diff looks good, other than unused default values for datadog and the serviceaccount ARN annotation (default "example-app"). Harmless, but you could switch them off if you really want to.

**Comment by danopia** [comment]
There's still an example-app in the yaml:

```
+     eks.amazonaws.com/role-arn: arn:aws:iam::[....]:role/kubernetes/cores/example-app
```

If we get that resolved then it'll LGTM

File: deployments/helm/helm-rollback-web/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -6,8 +6,8 @@ metadata:
   labels:
     {{- include "helmrollbackweb.labels" . | nindent 4 }}
   {{- with .Values.serviceAccount.annotations }}
-  annotations:
-    {{- toYaml . | nindent 4 }}
+  # annotations:
+  #   {{- toYaml . | nindent 4 }}
   {{- end }}
```
Comment (michael-robertson) [inline deployments/helm/helm-rollback-web/templates/serviceaccount.yaml:11]: standard chart creates the service account, this one should go away.

Comment (michael-robertson) [reply deployments/helm/helm-rollback-web/templates/serviceaccount.yaml:11]: (but we still need the clusterrolebinding)

Comment (michael-robertson) [reply deployments/helm/helm-rollback-web/templates/serviceaccount.yaml:11]: I recommend also renaming the file accordingly to avoid future confusion.

**Pull request merged**